### PR TITLE
Add hhfab vlab setup-vpcs helper

### PR DESCRIPTION
**Pull request by sergeymatov**

### CODE REVIEW

File: docs/vlab/demo.md [hunk 1]
```diff
@@ -192,6 +192,50 @@ core@server-02 ~ $ ip a
        valid_lft forever preferred_lft forever
 ```
 
+> **_NOTE_** Make usre that route table on servers is properly configured if you setting up interfaces manually.
```
Comment (Frostman) [inline docs/vlab/demo.md:195]: typo: usre -> sure and can you describe potentially most popular problem about default routes

File: docs/vlab/demo.md [hunk 2]
```diff
@@ -192,6 +192,50 @@ core@server-02 ~ $ ip a
...
+11:50:40 INF Configuring networking for server... server=server-08 netconf="bond 1008 enp2s1 enp2s2" conn=server-08--bundled--leaf-04
+11:51:06 INF Server network configured server=server-08 output=10.0.8.10/24 took=25.765623783s
+11:51:06 INF Configuring networking for server... server=server-09 netconf="vlan 1009 enp2s1" conn=server-09--unbundled--leaf-05
+11:51:16 INF Server network configured server=server-09 output=10.0.9.10/24 took=10.444237021s
+11:51:16 INF Configuring networking for server... server=server-10 netconf="bond 1010 enp2s1 enp2s2" conn=server-10--bundled--leaf-05
+11:51:31 INF Server network configured server=server-10 output=10.0.10.10/24 took=14.407390535s
+11:51:31 INF VPCs and VPCAttachments created, IP addresses discovered took=2m38.021452553s
+
+```
+In this example 10 VPC has been created for 10 servers. Subnets was autogenerated based on Ipv4namespace base CIDR. All servers get attachments based on Connections (mc-lag, es-lag, unbundled, etc).
```
Comment (Frostman) [inline docs/vlab/demo.md:237]: let's use spelling from our API: IPv4Namespace, MCLAG, ESLAG

File: docs/vlab/running.md [hunk 3]
```diff
@@ -168,6 +168,9 @@ Line `VM installed name=control-1` from the installer's output means that the in
 has been displayed, you can get into the control node and other VMs to watch the Fabric coming up and switches getting
 provisioned.
 
+> **_NOTE:_** SONiC system bootstrap may take reasonable time. It's heavily rely on the host disk and CPU performance, and parallel VM setup
```
Comment (Frostman) [inline docs/vlab/running.md:171]: it heavily relies on

**Review by Frostman — CHANGES_REQUESTED** [review]

File: docs/vlab/demo.md [hunk 4]
```diff
@@ -192,6 +192,50 @@ core@server-02 ~ $ ip a
        valid_lft forever preferred_lft forever
 ```
 
+> **_NOTE_** Make sure that route table on servers is properly configured if you setting up interfaces manually.
```
Comment (Frostman) [inline docs/vlab/demo.md:195]: Could you please describe here what is "properly configured" means

**Review by Frostman — CHANGES_REQUESTED** [review]

File: docs/vlab/demo.md (hunk 2 above)
Comment (mrbojangles3) [reply docs/vlab/demo.md:237]: Grammar nits:
"10 VPCs have been created" 
"Subnets have been auto generated"
"All servers are assigned attachments"

**Review by mrbojangles3 — CHANGES_REQUESTED** [review]
Minor Grammar Nits attached to Sergei's comments

File: docs/vlab/demo.md (hunk 2 above)
Comment (sergeymatov) [reply docs/vlab/demo.md]: done

**Review by mrbojangles3 — APPROVED** [review]

**Pull request closed without merging**